cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5401,6 +5401,22 @@
     </description>
   </property>
 
+  <property>
+    <name>tethering.client.connection.timeout.millis</name>
+    <value>5000</value>
+    <description>
+      Tethering client connection timeout in milliseconds.
+    </description>
+  </property>
+
+  <property>
+    <name>tethering.client.read.timeout.millis</name>
+    <value>20000</value>
```

Comment:
there are already configs for http.client.connection.timeout.ms and http.client.read.timeout.ms.

It would be good to keep the same naming (ms vs millis), but also good to default these to whatever is set for those (instead of 20000, set this to `${http.client.read.timeout.ms}`)